core/client/batch_client.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package client
+
+import (
+	"context"
+
+	"github.com/google/tink/go/tink"
+	"google.golang.org/grpc"
+
+	"github.com/google/keytransparency/core/mutator/entry"
+
+	tpb "github.com/google/keytransparency/core/api/type/type_go_proto"
+	pb "github.com/google/keytransparency/core/api/v1/keytransparency_go_proto"
+)
+
+// BatchCreateUser inserts mutations for new users that do not currently have entries.
+// Calling BatchCreate for a user that already exists will produce no change.
```

Comment:
Where on the stack does the "no change" happen? Can't see any entry skipping in this PR.

Reply:
This no change happens in the sequencer when it verifies `entry.Previous`.
`NewMutation` sets `entry.Previous` to `sha256.Sum(nil)`, which will only be valid if the user does not already exist.